### PR TITLE
Update 1.21.11 changelog

### PR DESCRIPTION
**Pull request by lunny**
Add #30394 into changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ been added to each release, please refer to the [blog](https://blog.gitea.com).
   * Fix incorrect SVGs (#30087)
   * Fix create commit status (#30225) #30340
   * Performance optimization for git push (#30104) #30354
+  * Fix Inconsistency accessing user's variables and secrets #30402
```
Comment (wolfogre) [inline CHANGELOG.md]:
```suggestion
  * Fix create commit status (#30225) (#30340)
  * Performance optimization for git push (#30104) (#30354)
  * Fix misuse of unsupported global variables (#30402)
```

**Review by wolfogre — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]

**Comment by yp05327** [comment]
image

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,15 +16,30 @@ been added to each release, please refer to the [blog](https://blog.gitea.com).
...
+  * Fix commit status cache which missed target_url (#30426)
+  * Fix rename branch 500 when the target branch is deleted but exist in database (#30430)
+  * Fix mirror error when mirror repo is empty (#30432)
+  * Use db.ListOptions directly instead of Paginator interface to make it easier to use and fix performance of /pulls and /issues (#29990)
+  * Fix code owners will not be mentioned when a pull request comes from a forked repository #30476 
+* DOCS
+  * Update actions variables documents (#30394) #30405
 * MISC
   * Update katex to 0.16.10 (#30089)
+  * Upgrade go-sqlite to v1.14.22
```
Comment (yp05327) [inline CHANGELOG.md]:
```suggestion
  * Upgrade go-sqlite to v1.14.22 (#30462)
```

**Comment by yp05327** [comment]
It seems that newly added changes have no original PR id

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,15 +16,30 @@ been added to each release, please refer to the [blog](https://blog.gitea.com).
...
+  * Performance optimization for git push (#30104) (#30354)
+  * Fix misuse of unsupported global variables (#30402)
+  * Fix to delete the cookie when AppSubURL is non-empty (#30375) (#30468)
+  * Avoid user does not exist error when detecting schedule actions when the commit author is an external user (#30357) (#30408)
+  * Change the default maxPerPage for gitbucket (#30392) (#30471)
+  * Check the token's owner and repository when registering a runner (#30406) (#30412)
+  * Avoid losing token when updating mirror settings (#30429) (#30466)
+  * Fix commit status cache which missed target_url (#30426) (#30445)
+  * Fix rename branch 500 when the target branch is deleted but exist in database (#30430) (#30437)
+  * Fix mirror error when mirror repo is empty (#30432) (30467)
```
Comment (yp05327) [inline CHANGELOG.md]:
```suggestion
  * Fix mirror error when mirror repo is empty (#30432) (#30467)
```

**Pull request merged**